src/types/ical.js.d.ts

```diff
@@ -43,7 +43,7 @@ declare module 'ical.js' {
 
     public component: Component;
 
-    public constructor(component?: Component | null, options?: {strictExceptions: boolean, exepctions: Array<Component | Event>});
+    public constructor(component?: Component | null, options?: { strictExceptions: boolean, exepctions: Array<Component | Event> });
```

Comment:
All of the changes to this file should be removed, please don't mix style and logic changes in a PR, it makes things very hard to review.